packages/salesforcedx-vscode-soql/src/commands/soqlFileCreate.ts

```diff
@@ -17,16 +17,16 @@ export async function soqlOpenNew(): Promise<void> {
   telemetryService.sendCommandEvent('soql_builder_open_new', process.hrtime());
 
   if (vscode.workspace) {
-    // create untitled file
-    const doc = await vscode.workspace.openTextDocument({
-      language: 'soql',
-      content: ''
+    const fileName = 'untitled.soql';
+    const newUri = vscode.Uri.file(fileName).with({
+      scheme: 'untitled',
+      path: fileName
```

Comment:
ultimately, the Uri for an untitled file ends up looking like `untitled:untitled.soql` instead of when the file is on disc, which is a `file:` scheme.